### PR TITLE
Add section for AstroNvim configuration

### PR DESCRIPTION
**Pull request by KhudhurM**
AstroNvim is a feature-rich neovim config for beginners. It makes neovim approachable for many beginners such as me and it uses nvim-lspconfig under the hood.

I struggled with porting the "known issue" fix from nvim-lspconfig to the existing AstroNvim configuration, since it differs in structure / config file location.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -50,7 +50,6 @@ require'lspconfig'.cssmodules_ls.setup {
 ```
 
 ### [coc.nvim](https://github.com/neoclide/coc.nvim)
-
```
Comment (antonk52) [inline README.md:53]: This change seem unrelated

File: README.md [hunk 2]
```diff
@@ -62,6 +61,31 @@ let cssmodules_config = {
 coc#config('languageserver.cssmodules', cssmodules_config)
 ```
 
+### [AstroNvim](https://github.com/AstroNvim/AstroNvim)
+As per [`AstroNvim's documentation`](https://astronvim.github.io/#%EF%B8%8F-installation), you can install cssmodules_ls with:
```
Comment (antonk52) [inline README.md:65]:
```suggestion
### [AstroNvim](https://github.com/AstroNvim/AstroNvim)

As per [`AstroNvim's documentation`](https://astronvim.github.io/#%EF%B8%8F-installation), you can install cssmodules_ls with:
```

**Review by antonk52 — COMMENTED** [review]
Hi and thank you for the PR. I am happy to merge it, just a couple of cosmetic remarks.

File: README.md (hunk 1 above)
Comment (KhudhurM) [reply README.md]: Reverted this in new commit, thanks for considering merging!
